website/src/css/_navbar.scss

```diff
@@ -22,3 +22,29 @@
   padding: 5px;
   font-weight: 300;
 }
+/*===========*/
+
+/* Restyle gamburger button */
+
+.navbar__toggle {
+  width: 40px;
+  height: 40px;
+  background-color: var(--detox-color-a3);
+  border-radius: 10px;
+  justify-content: center;
+  align-items: center;
+}
+
+/* Restyle search button */
+
+@media (max-width: 769px) {
+  button.DocSearch-Button {
+    background-color: var(--detox-color-a3);
+    height: 40px;
+    border-radius: 10px;
+    padding: 0 10px;
+  }
+  button.DocSearch-Button svg {
+    opacity: 0.6;
+  }
+}
```

Comment:
I reviewed your changes visually: https://percy.io/66fcbcad/detox-website/builds/25010682/changed/1396573909

Unfortunately, you forgot to override the positioning of `.navbar__toggle` for mobile viewports. Please add `position: absolute` override and pick a correct value for `right: ??px`.

| Actual | Expected |
|--------|----------|
| ![](https://user-images.githubusercontent.com/1962469/218054895-6ae0248e-3fc9-4cdd-83fe-176cedd0724c.png) | ![](https://user-images.githubusercontent.com/1962469/218054846-c96bca14-a92d-48d8-9940-5aa8d1995041.png) |